app/com/arpnetworking/metrics/portal/alerts/impl/DailyPartitionCreator.java

```diff
@@ -205,6 +206,11 @@ private void recordCounter(final String metricName, final long value) {
         _periodicMetrics.recordCounter(fullMetric, value);
     }
 
+    private void recordTimer(final String metricName, final Duration duration) {
+        final String fullMetric = String.format("partition_creator/%s/%s", _table, metricName);
+        _periodicMetrics.recordTimer(fullMetric, duration.toMillis(), Optional.of(TimeUnit.MILLISECONDS));
```

Comment:
There are some timeout errors when calling `DailyPartitionCreator.ensurePartitionExists` which is done via `Patterns.ask` to the actor managing the partitions. I added this to see if the operation is really taking longer than a second, or if the actor goes missing on the host.